my-cypress-project/cypress/pages/realWordAppPage.js

```diff
@@ -0,0 +1,117 @@
+export default class LoginPage {
+    constructor() {
+        this.userInfo = {
+            firstName: "Olena",
+            lastName: "Gofman",
+            username: "OlenaGofman",
+            password: "secret",
+            confirmPassword: "secret"
+        }
+
+        this.date = new Date().getTime();
+    }
+    openRealWorldApp() {
+        cy.visit('http://localhost:3000/')
+
+    }
+    signUp() {
```

Comment:
Tricky action. In theory we should return the state of the system to the same, as before the test.
In real life it requires deleting  created users, what technically is not implemented in real systems (each user stays in the DB forever, just marked as deleted).
It is recommended to agree with developers about such cases.